specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,3232 @@
+{
+  "swagger": "2.0",
```

Comment:
Do we need to maintain both swagger and typespec specifications going forward? Even the examples are duplicated.